core/App/ffi_operation.cpp

```diff
@@ -275,7 +275,7 @@ extern "C"
             goto out;
         }
 
-        if (master_key_tmp.keybloblen == 0 || master_key_tmp.keybloblen > UINT16_MAX)
+        if ((master_key_tmp.keybloblen == 0 && master_key_tmp.metadata.origin != EH_EXTERNAL_KEY) || master_key_tmp.keybloblen > UINT16_MAX)
         {
```

Comment:
all the key need to set as EH_EXTERNAL_KEY?

Reply:
yes,only keybloblen of external key is 0.

Reply:
yes